pysoa/common/logging.py

```diff
@@ -188,6 +188,7 @@ class RecursivelyCensoredDictWrapper(object):
         'personal_id_numbers', 'personal-id-numbers', 'personalIdNumbers',
         'personal_identification_number', 'personal-identification-number', 'personalIdentificationNumber',
         'personal_identification_numbers', 'personal-identification-numbers', 'personalIdentificationNumbers',
+        'tin_number', 'tin-number', 'tinNumber',
```

Comment:
`tinNumber` is akin to `ATM Machine` (Automatic Teller Machine Machine).  I'd recommend we just spell it out like we do with personal identification number?